Source/Fuse.Controls.Native/Android/SystemScreenConfigAndroidImpl.uno

```diff
@@ -0,0 +1,104 @@
+using Fuse.Controls.Native;
+
+namespace Fuse.Controls.Native
+{
+	extern(Android) internal class SystemScreenConfigAndroidImpl
+	{
+		private static SystemUiVisibility.Flag _wantedFlags = SystemUiVisibility.Flag.None;
+		private static SystemScreenConfig config;
+
+		public static void init(SystemScreenConfig config)
```

Comment:
methods have names that start with upper-case letters in fuselibs, so this should be "Init", not "init".